app/src/main/java/com/wednesday/banklocator/db/IfscDao.kt

```diff
@@ -0,0 +1,19 @@
+package com.wednesday.banklocator.db
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+import com.wednesday.banklocator.model.IfscResponse
+
+
+@Dao
+interface IfscDao {
```

Comment:
When I read this interface it feels like the database and the network objects are tightly coupled. Maybe it's just the name of the class. For e.g.

`suspend fun upsert(Ifsc: IfscResponse): Long`

should be

`suspend fun upsert(ifsc: Ifsc): Long`

Reply:
Okay, I'll fix it!